common/doc_store/es_conn_pool.py

```
@@ -31,16 +33,30 @@ def __init__(self):
+                    self.ES_CONFIG["hosts"],
+                    attempt + 1,
+                    MAX_RETRIES,
+                    e,
+                )
+                if attempt == MAX_RETRIES - 1:
+                    raise
+                if hasattr(self, "es_conn") and self.es_conn:
+                    try:
+                        self.es_conn.close()
+                    except Exception:
+                        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)